scripts/profiler_parser.py

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env python
+
+"""
+Utility script to parse output from Profiler framework and print statistics.
```

Comment:
Please add a pointer to the documentation, so that who reads this script can be aware that the documentation has a full section for explaining what it does. Thanks!

Reply:
I thought that the documentation concerning the script should have been inserted inside the script itself, in the initial lines. If I am wrong, tomorrow I am going to correct it adding a specific section in the Project documentation.
Let me know.